### ISSUE TITLE
`pyflow init` fails

### ISSUE THREAD

**Issue by eduidl** [David-OConnor/pyflow#43]
Hello.
I want to initialize pyflow project from existing Pipfile, however `pyflow init` fails even in empty directory. How can I do?

```terminal
$ mkdir test
$ cd test
$ pyflow init
To get started, run `pyflow new projname` to create a project folder, or `pyflow init` to start a project in this folder. For a list of what you can do, run `pyflow help`.
```

My environment is as follows.

- Ubuntu 18.04
- cargo: 1.39.0
- pyflow: 0.2.3 (installed by cargo)

Regards.

**Comment by David-OConnor** [comment]
Fixed in the latest commit. Note that the Pipfile parsing code was also pretty bad; that should be fixed as well. Expect a new release soon.

**Comment by eduidl** [comment]
Thanks for your reply. I'm looking forward to a next release!

**Comment by David-OConnor** [comment]
Released as 0.2.4.
